### DISCUSSION TITLE
Other audio stops when a notification sounds

### DISCUSSION THREAD

**Discussion by synstin** [gdelataillade/alarm#120] [General]
@gdelataillade

https://github.com/gdelataillade/alarm/issues/100

Are there any plans to fix this issue in a future version?
There are too many complaints from users.
When a notification goes off for a short task, the music stops, which is really bad for the user experience.

Let us know if you plan to fix it :)
Thank you !

**Comment by gdelataillade** [comment]
@synstin

I just answered in the discussion #104
